skimage/setup.py

```diff
@@ -18,6 +18,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('graph')
     config.add_subpackage('io')
     config.add_subpackage('measure')
+    config.add_subpackage('metrics')
```

Comment:
@jni, is this necessary? Because I just noticed that the `registration` subpackage is not added to config :/ ...

Reply:
This is needed when there is Cython to compile, otherwise it is not needed, but doesn't hurt anything.